### PR TITLE
fix for 196: create a uber-jar as a workaround for maven's class loading problem (MNG-5787)

### PR DESCRIPTION
**Pull request by TheSnoozer**
currently only need to include org.slf4j in the uber-jar

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -212,6 +212,33 @@
           <target>${java.target}</target>
         </configuration>
       </plugin>
+      <!-- we need to create a uber war for https://github.com/ktoso/maven-git-commit-id-plugin/issues/196 -->
```
Comment (ktoso) [inline pom.xml]: uber jar :)

**Comment by ktoso** [comment]
If it works LGTM, makes sense to jarjar it

File: pom.xml (hunk 1 above)
Comment (TheSnoozer) [inline pom.xml]: fixed....think before typing :-)

**Comment by ktoso** [comment]
Thanks!

**Comment by TheSnoozer** [comment]
always welcome :-)

**Pull request merged**